lessons/operator/01/README.md

```diff
@@ -1,122 +1,121 @@
+Sensu provides an _integrated_ pipeline with a _consistent_ data model. 
+Operating like a control-plane, Sensu gives you a flexible means by which to direct multiple streams of observations and metrics to whatever destination you need them to go, and transforms them into whatever shape they need to be along the way.
 
-If the pseudo architecture diagram (or "marketecture diagram") looks a little bit like a [Rube Goldberg machine](https://en.wikipedia.org/wiki/Rube_Goldberg_machine), that's effectively what all DIY observabilty pipelines become, sooner or later.
-Thankfully, you don't need to cobble together collection of services to adopt an observability pipeline – Sensu Go is a turn-key observabilty pipeline that enables monitoring as code on any cloud.
+It can do this because every observation in Sensu is contained in a single flexible data structure; the _event_.
 
-![](img/sensu-observability-pipeline.png)
+**Example: A Sensu Event Data Structure**
+```json
+{
+  "metadata": {},
+  "entity": {},
+  "check": {},
+  "metrics": {},
+  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+  "timestamp": 1234567890
+}
+```
 
-_FIGURE 2: Sensu's turn-key solution enables organizations to adopt observability pipelines with ease._
+Events can contain numeric metrics, dictionaries of metadata, labels, and annotations, as well as any other payload of information that is meaningful to your system.
+Each component in the pipeline expects a stream of events as input and outputs a stream to the next component.
 
-## Why Sensu?
+Learn more about the event data structure in the [events reference docs](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/)
 
-What does "monitoring as code" and the "observability pipeline" have to do with eachother?
-And how is Sensu's solution better than the alternatives (including the DIY pipeline)?
-This question is answered in our [monitoring as code whitepaper](https://sensu.io/resources/whitepaper/monitoring-as-code-with-sensu) (bold added for emphasis):
+## Monitoring as Code
 
-> Everything as code initiatives as we know them are built on top of **pipelines that enable organizations to codify complex business requirements as repeatable steps**.
-> The CI pipeline (build+test) paved the way for Continuous Delivery (CD).
-> Together as the CI/CD pipeline (build+test, deploy), they provided the foundation for infrastructure as code (build+test, provision+test, deploy).
-> Like IaC, monitoring as code hooks into this CI/CD pipeline, but also requires a dedicated pipeline for ongoing observation of our systems (build+test, provision+test, deploy, observe).
-> We call this missing link the observability pipeline.
->
-> ![](img/everything-as-code.png)
+There is a strong movement towards managing all aspects of the software industry in the same way that we manage application code. 
```

Comment:
s/the software industry/IT operations/ ??

Reply:
This is an interesting question ... While IT Ops may be more of our target customers, this terminology would narrow things down to exclude non-IT Ops folks. Also, it limits the range of concept we're talking about here. The "as-code" movement is across *all* job functions in the software industry, including IT Ops. If we say there's a strong movement in IT Ops, while that's true, it doesn't provide tie-in to those other movements, and works against the "DevOps" mentality that the lines between developer and operator are increasingly blurry and that's a good thing. Specifically, that's a thing that "as-code" movements enable and encourage.